BlueprintUICommonControls/Sources/Box.swift

```diff
@@ -110,8 +116,10 @@ extension Box {
 
     public enum CornerStyle {
         case square
-        case capsule
-        case rounded(radius: CGFloat, corners: Corners = .all)
+        case capsule(curve: CornerCurve = .circular)
+        case rounded(radius: CGFloat, corners: Corners = .all, curve: CornerCurve = .circular)
```

Comment:
The defaults here keep the old standard, however `.continuous` definitely looks more "iOS-y"; thoughts on swapping the defaults? Would affect POS snapshots.

Reply:
It's unfortunate that this would be a breaking change.. maybe we should add a:
```swift
@deprecated(...)
static var capsule: CornerStyle {
    .capsule()
}
```
to assist with migration?

Reply:
Ah, missed that.

Should we deprecate the non `()` version or something?

I got some push back on adding conveniences like this in the past. I _think_ it was @watt, so curious he has thoughts.

Reply:
To me the deprecation doesn't add a lot of value; just more noise unless we explicitly migrate everyone every time we add cases to enums; so I'd keep both.

Another option I was considering was adding `cornerCurve` as a top level property on `Box`, so it doesn't need to be in the enum. Thoughts?

Reply:
> I got some push back on adding conveniences like this in the past. I _think_ it was @watt, so curious he has thoughts. I think I do like having these whenever there are ideal defaults in the parameter list.

Could have been me. I dislike the ambiguity of having both `capsule` and `capsule()` options, especially undocumented. The parens let you know there's an associated value — you can't call a function `foo(bar: Int = 0)` as `foo` either.  I won't block on it though.

> The defaults here keep the old standard, however .continuous definitely looks more "iOS-y"; thoughts on swapping the defaults? Would affect POS snapshots.

I think we should keep the circular default. This is particularly important for `capsule`, which I think is primarily used to make circles.

> Another option I was considering was adding cornerCurve as a top level property on Box, so it doesn't need to be in the enum. Thoughts?

I'd be OK with this. We have a [flattened representation in Modals](https://github.com/squareup/market/blob/main/ios/Modals/Sources/ModalRoundedCorners.swift).

Reply:
I'm going to move this up to the `Box` root then, I think it's nicer to not keep making these enums longer and longer; and solves the source-breaking `()` addition